public/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "requires": {
+    "cockpit": "134"
+  },
+  "version": 0,
+  "name": "composer",
```

Comment:
Decide what URL you want to use. If it's images in the path you want to change that.